cmd/ocm/create/cluster/cmd.go

```diff
@@ -301,6 +313,16 @@ func init() {
 		fmt.Sprintf("Namespace Ownership Policy for ingress. Options are %s",
 			strings.Join(ingress.ValidNamespaceOwnershipPolicies, ",")),
 	)
+
+	fs.StringVar(
+		&args.subscriptionType,
+		"subscription-type",
+		billing.StandardSubscriptionType,
+		fmt.Sprintf("The subscription billing model for the cluster. Options are %s",
```

Comment:
missing punctuation mark

Reply:
Keeping the options description consistent with https://github.com/openshift-online/ocm-cli/blob/main/cmd/ocm/create/cluster/cmd.go#L294 and https://github.com/openshift-online/ocm-cli/blob/main/cmd/ocm/create/cluster/cmd.go#L301